### PR TITLE
Move out hard-coded resources

### PR DESCRIPTION
**Pull request by Jakob37**
# Description

Move out hard-coded resources from main.nf

## Type of change

- [x] Patch

# Checklist:

- [x] My code follows the [style guidelines for NF pipelines at CMD](http://mtlucmds1.lund.skane.se/wiki/doku.php?id=nextflow&s[]=nextflow#code_style_at_cmd)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings (Keep an eye on `.nextflow.log` !)
- [x] I have updated the CHANGELOG
- [x] The latest commit in the master branch is tagged
      with the correct version number

## Patch

- [x] Stub run completes without errors or new warnings

Final changes

- [x] Update CHANGELOG with moved annotation fields
- [x] Do manual check that annotations haven't changed by the updates

# Post-merge

- [x] The `master` branch has been tagged with the new version number.

### CODE REVIEW

File: configs/nextflow.hopper.config [hunk 1]
```diff
@@ -37,49 +37,48 @@ params {
...
-  panelsdef = "/fs1/resources/scout/scout_gene_panel__latest.json"
+  scoutbed          = "${refpath}/bed/scout/hgnc.grch38p13.exons.sort.bed"
+  panelsdef         = "/fs1/resources/scout/scout_gene_panel__latest.json"
 
-  gene_regions = "${refpath}/gtf/gencode.v33.exons.sort.merge.bed"
+  gene_regions      = "${refpath}/gtf/gencode.v33.exons.sort.merge.bed"
   
   // EXPANSION HUNTER //
-  expansionhunter_catalog = "${refpath}/expansionhunter/variant_catalog_hg38_5.0.0.json"
+  expansionhunter_catalog     = "${refpath}/expansionhunter/variant_catalog_hg38_5.0.0.json"
```
Comment (alkc) [inline configs/nextflow.hopper.config:70]: ❤️‍🔥

File: main.nf [hunk 2]
```diff
@@ -1852,12 +1852,12 @@ process annotate_vep {
...
 			--plugin MaxEntScan,$params.MAXENTSCAN,SWA,NCSS \\
-			--plugin dbNSFP,/fs1/resources/ref/hg38/annotation_dbs/dbnsfp/dbNSFP4.3a_grch38.gz,transcript_match=1,REVEL_score,REVEL_rankscore \\
+			--plugin dbNSFP,$params.DBNSFP,transcript_match=1,REVEL_score,REVEL_rankscore \\
 			-custom $params.GNOMAD_EXOMES,gnomADe,vcf,exact,0,AF_popmax,AF,popmax \\
 			-custom $params.GNOMAD_GENOMES,gnomADg,vcf,exact,0,AF_popmax,AF,popmax \\
 			-custom $params.GNOMAD_MT,gnomAD_mt,vcf,exact,0,AF_hom,AF_het \\
-			-custom $params.PHYLOP \\
-			-custom $params.PHASTCONS
+			-custom $params.PHYLOP,phyloP100way,bigwig \\
+			-custom $params.PHASTCONS,phastCons,bigwig
```
Comment (alkc) [inline main.nf:1860]: What are those two changes? Should they be documented in the changelog?

**Review by alkc — APPROVED** [review]

File: main.nf (hunk 2 above)
Comment (Jakob37) [reply main.nf:1860]: These where previously defined in the config file, i.e. part of the "$params.PHASTCONS" variable. So the main.nf + nextflow.conf pair should produce the same results after this update, if done correctly.

But yes, makes sense to specify this change in the changelog

Comment (alkc) [reply main.nf:1860]: Ohh. I see it now.

**Pull request merged**